.github/workflows/browserstack-workflow.yml

```diff
@@ -0,0 +1,74 @@
+name: BrowserStack
+
+# BrowserStack tests are run against the deployed GitHub pages URL
+# The closest event we can use to trigger when these are updated is page_build.
+# on: [page_build]
+on: [push]
```

Comment:
@davidrapson  we want this to on page build, right?

Reply:
Oh good point. Yeah that's the ideal event I think. A build scoped to `master` is fine as a fallback if `page_build` doesn't do what we think but in theory it's closest to the thing we're testing in BrowserStack.

Reply:
Cool, I just checked the docs again and it's a push to any github pages enabled branch so ✅